source/ddbus/thin.d

```diff
@@ -27,6 +26,19 @@ struct ObjectPath {
     return _value;
   }
 
+  /// Returns the string representation of this ObjectPath
+  string value() const pure nothrow @safe {
```

Comment:
Maybe just call it `toString`?

Reply:
there is toString which right now does exactly this, but it has less attributes (in case it will get changed and include something like prepending `"Object Path"` because toString is used to represent the object in a human readable way. Adding a value method allows to modify toString in the future + this has documentation and is guaranteed to be what the developer is looking for (instead of trying toString and hoping it won't break)